src/app/detail/ui/TaskEditor.tsx

```diff
@@ -170,6 +170,11 @@ export const TaskEditor = ({
             disabled={!isEditable}
             padding="0px"
             onBlur={handleTitleBlur}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter') {
```

Comment:
we have `multiline` attribute in StyledTextField, is this what we need if we dont want multiple line? Or is it just for longer length title?

Reply:
We want multiple line, but only automatically(not manually). this prevents users from breaking line manually. 

For example, in this github comment box we can write text with multiline. 
ALSO 
LIKE 
THIS 
TOO which looks bad in task title IMO.